content/francais/python-turtle/the-basics.md

```diff
@@ -0,0 +1,88 @@
+
+Tout d'abord, nous devons dire à la tortue quelle couleur nous voulons que la ligne soit. Pour cela, tapons :
+
+``` python
+turtle.color("orange")
+```
+
+Essayons de comprendre cette ligne de code plus en détail.
+
+{{% notice tip %}}
+
+## Chaînes de caractères
+
+Tout ce que vous voyez entre **guillemets** s'appelle une **chaîne de caractères**. Une chaîne est simplement une séquence de caractères (qu'il s'agisse de lettres, de chiffres ou de symboles).
+
+Dans le code ci-dessus, `"orange"` est une chaîne de caractères. D'autres exemples de chaînes incluent `"123"`, `"abc!"` et `"green"`. Attention, `123`, `abc!` et `green` ne sont pas des chaînes car il manque des guillemets !
+
+## Méthodes
+
+`turtle.color()` est une **méthode.** Nous utilisons des méthodes pour que la tortue effectue certaines actions. Dans ce cas, `turtle.color()` change la couleur des lignes qu'elle dessine selon la couleur que nous lui indiquons. Nous lui avons dit de changer la couleur en orange en lui envoyant la **chaîne** `"orange"` comme **argument**. 
+
+## Arguments
+
+Les arguments sont des entrées que vous passez aux **méthodes** pour utiliser. Dans notre exemple, `turtle.color()` a besoin d'un argument de type **chaîne** représentant le nom de la couleur à définir pour la tortue, donc nous avons passé `"orange"` comme **argument** à la méthode. Nous aurions aussi pu lui dire de changer la couleur en quelque chose d'autre, mais il fallait que ce soit une couleur que `turtle.color()` comprend. Les couleurs qu'il reconnaît sont : `"red"`, `"orange"`, `"yellow"`, `"green"`, `"blue"`, `"purple"`, `"black"`, `"pink"`, et `"gray"`. Si nous envoyons autre chose qu'une de ces chaînes de couleurs, nous obtenons la couleur par défaut, qui est le noir.
+
+{{% /notice %}}
+
+Maintenant que nous avons choisi la couleur que nous voulons, nous pouvons maintenant dessiner une ligne ! Pour dessiner une ligne, il y a deux **méthodes** que nous pouvons utiliser :
+
+``` python
+turtle.forward(50)
+turtle.backward(50)
+```
+
+`turtle.forward(50)` fera avancer la tortue de 50 pixels et dessinera une ligne dans la direction où elle est orientée, tandis que `turtle.backward(50)` fera la même chose, mais dans la direction opposée.
+
+{{% notice tip %}}
+
+## Pixels
+
+Une image est composée de milliers, voire de millions de pixels. Un pixel est une unité de mesure pour les images numériques, similaire à la façon dont nous mesurons le poids en livres, ou les longues distances en miles.
+
+## Entiers
+
+Un entier est simplement n'importe quel nombre entier, qu'il soit positif ou négatif. Dans cet exemple, `turtle.forward()` et `turtle.backward()` attendent tous deux un **entier** comme argument plutôt qu'une chaîne. C'est parce que l'argument représente le nombre de pixels que vous voulez que la tortue se déplace lors du dessin. Une chaîne ne fonctionnerait évidemment pas pour ces méthodes !
```

Comment:
this is a tricky edit, so will add the change as a whole:

Un **integer** est simplement n'importe quel nombre entier, qu'il soit positif ou négatif. Dans cet exemple, `turtle.forward()` et `turtle.backward()` attendent tous deux un **integer** comme argument plutôt qu'une **string**. C'est parce que l'argument représente le nombre de pixels que vous voulez que la tortue se déplace lors du dessin. Une **string** ne fonctionnerait évidemment pas pour ces méthodes !